index.html

```diff
@@ -128,39 +161,44 @@ <h3>Skills</h3>
             <div class="contact-container">
                 <h2>Get started</h2>
                 <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
-                sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+                sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+                Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris 
+                nisi ut aliquip ex ea commodo consequat.</p>
                 <form action="https://formspree.io/f/mqkwbzjb" method="post">
                     <div class="form-fields">
                      <div class="float-container">
-                       <span for="name">Your name</span>
+                       <span>Your name</span>
                        <input type="text" id="name" name="user_name" placeholder="Jhon Doe" maxlength="30" required>
                      </div>
                      <div class="float-container">
-                       <span for="mail">E-mail</span>
+                       <span>E-mail</span>
                        <input type="email" id="mail" name="user_email" placeholder="example@domain.com" required>
                      </div>
                      <div class="float-textarea">
-                       <span for="msg"></span>
                        <textarea id="msg" name="user_message" placeholder="Enter your message here..." maxlength="500" required></textarea>
                      </div>
                     </div>
-                    <button>Start collaboration</button>
                 </form>
+                <button>Start collaboration</button>
             </div>
         </section>
         <footer>
             <div class="get-resume">
                 <a>Get my Resume</a>
                 <img src="img/ic_link_teal.svg" alt="link-teal" width="22">
-            </div>            
-            <div class="networks">
-                <img src="./img/ic_stackoverflow.svg" alt="">
-                <img src="./img/ic_github.svg" alt="">
-                <img src="./img/ic_twitter.svg" alt="">
-                <img src="./img/ic_linkedin.svg" alt="">
-                <img src="./img/ic_medium.svg" alt="">
-                <img src="./img/ic_angellist.svg" alt="">
             </div>
+            <div class="get-cv">                
+                    <span>Get my CV</span>
+                    <img src="img/download.png" alt="download" width="22">
+            </div>
+            <ul class="networks">
+                <li><img src="./img/ic_stackoverflow.svg" alt=""></li>
+                <li><img src="./img/ic_github.svg" alt=""></li>
+                <li><img src="./img/ic_twitter.svg" alt=""></li>
+                <li><img src="./img/ic_linkedin.svg" alt=""></li>
+                <li><img src="./img/ic_medium.svg" alt=""></li>
+                <li><img src="./img/ic_angellist.svg" alt=""></li>
+            </ul>
         </footer>
```

Comment:
Please style this section as the original one shown in the screenshot below
![right3](https://user-images.githubusercontent.com/58376020/121606052-c4530600-ca4d-11eb-8682-874e1d4ec303.png)

## Current style
![wrong3](https://user-images.githubusercontent.com/58376020/121606084-d0d75e80-ca4d-11eb-8153-5e844d12fb65.png)

Reply:
done 👍